### PR TITLE
Dev-12_BU_Wordsearch

### PR DESCRIPTION
**Pull request by beccamupton**

### CODE REVIEW

File: Assets/Scripts/SearchingWord.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+using System.Collections;
```
Comment (cubastanley) [inline Assets/Scripts/SearchingWord.cs]: Think you can delete this script?

File: Assets/Scripts/SearchingWordsList.cs [hunk 2]
```diff
@@ -0,0 +1,183 @@
+using System.Collections;
```
Comment (cubastanley) [inline Assets/Scripts/SearchingWordsList.cs:1]: Should probably delete this script

**Review by cubastanley — CHANGES_REQUESTED** [review]
Just some cosmetic change requests - really solid though, I think I'll need a hand from you with integrating this with the game levels definitely 😂 may need to refactor a couple of bits for that but we'll cross that bridge when we get to it

File: Assets/Scripts/WordsGrid.cs [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+        // if statement for start position of grid
+        startPosition.x = (midWidthPosition != 0) ? midWidthPosition * -1 : midWidthPosition;
+        startPosition.y = midHeightPosition;
+
+        // check if grid is still in screen
+        return startPosition.x < GetHelfScreenWidth() * -1 || startPosition.y > topPosition;
+
+    }
+
+    private float GetHelfScreenWidth()
```
Comment (cubastanley) [inline Assets/Scripts/WordsGrid.cs]: Im assuming this one is a typo?

Comment (beccamupton) [reply Assets/Scripts/WordsGrid.cs]: haha yes

**Review by cubastanley — APPROVED** [review]

**Pull request merged**